src/util/customTransforrmationsStore-v1.js

```diff
@@ -64,4 +66,30 @@ async function getLibraryCodeV1(versionId) {
   }
 }
 
-module.exports = { getTransformationCodeV1, getLibraryCodeV1 };
+async function getRudderLibraryCode(importName) {
+  const rudderLibrary = rudderLibraryCache[importName];
+  if (rudderLibrary) return rudderLibrary;
+  const tags = {
+    libraryVersionId: importName,
+    version: 1,
+    type: 'rudderlibrary',
+  };
+  try {
+    const url = `${getRudderLibrariesUrl}/name?importName=${importName}`;
+    const startTime = new Date();
+    const response = await fetchWithProxy(url);
+
+    responseStatusHandler(response.status, 'Rudder Library', importName, url);
+    stats.increment('get_libraries_code.success', tags);
+    stats.timing('get_libraries_code', startTime, tags);
+    const myJson = await response.json();
+    rudderLibraryCache[importName] = myJson;
```

Comment:
I would name the response as `const code` instead of `myJson` for better naming. Wdyt ?

Reply:
it is library entity which contains code